test/resources/diff_generation/diff_generation.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: policy-diff-gen-configmap
+  annotations:
+    policy.open-cluster-management.io/standards: NIST SP 800-53
+    policy.open-cluster-management.io/categories: CM Configuration Management
+    policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+spec:
+  disabled: false
+  policy-templates:
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: policy-diff-gen-configmap
+        spec:
+          remediationAction: inform
+          severity: low
+          object-templates:
+            - complianceType: musthave
+              recordDiff: Log
+              objectDefinition:
+                apiVersion: v1
+                kind: ConfigMap
+                metadata:
+                  name: diff-gen-configmap
+                  namespace: default
+                data:
+                  fish: marlin
+                  cephalopod: squid
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: PlacementBinding
+metadata:
+  name: binding-policy-diff-gen-configmap
+placementRef:
+  name: placement-policy-diff-gen-configmap
+  kind: PlacementRule
+  apiGroup: apps.open-cluster-management.io
+subjects:
+- name: policy-diff-gen-configmap
+  kind: Policy
+  apiGroup: policy.open-cluster-management.io
+---
+apiVersion: apps.open-cluster-management.io/v1
+kind: PlacementRule
```

Comment:
Should this be a `Placement`? Probably. But I felt like implementing that was outside the scope of this PR.

Reply:
Yes, maybe we need a between release work item to make tests with Placement easy to add.

Reply:
Agreed. It should be straightforward in the end, especially if we wipe placement from the policy-collection and then have a common function to deploy placement for everything, but it'll be a bit of work to clean it all up!

Reply:
That said, maybe it'd all get wrapped into this issue? https://issues.redhat.com/browse/ACM-9081